custom/icds_reports/utils/data_pull_ap_rj.sql

```diff
@@ -0,0 +1,271 @@
+COPY(SELECT
+    state_name,
+    district_name,
+    block_name,
+    supervisor_name,
+    awc_name,
+    awc_site_code,
+    CASE WHEN SUM(CASE WHEN month='2019-04-01'  THEN thr_eligible ELSE 0 END)>0 THEN round((sum(CASE WHEN num_rations_distributed>=21 AND month='2019-04-01'  THEN 1 ELSE 0 END)/SUM(CASE WHEN month='2019-04-01'  THEN thr_eligible ELSE 0 END)::float)::numeric,2)*100 ELSE 0 END as pw_thr_21_April,
+    CASE WHEN SUM(CASE WHEN month='2019-05-01'  THEN thr_eligible ELSE 0 END)>0 THEN round((sum(CASE WHEN num_rations_distributed>=21 AND month='2019-05-01'  THEN 1 ELSE 0 END)/SUM(CASE WHEN month='2019-05-01'  THEN thr_eligible ELSE 0 END)::float)::numeric,2)*100 ELSE 0  END as pw_thr_21_May,
+    CASE WHEN SUM(CASE WHEN month='2019-06-01'  THEN thr_eligible ELSE 0 END)>0 THEN round((sum(CASE WHEN num_rations_distributed>=21 AND month='2019-06-01'  THEN 1 ELSE 0 END)/SUM(CASE WHEN month='2019-06-01'  THEN thr_eligible ELSE 0 END)::float)::numeric,2)*100 ELSE 0  END as pw_thr_21_June,
```

Comment:
Isnt this info all available in agg_ccs_record

Reply:
Yeah. But last time we did data pull. ccs_records were faster. May be because its distribuetd